Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -740,7 +740,7 @@ ShaderCode GeneratePixelShaderCode(APIType ApiType, const ShaderHostConfig& host
   }
   else
   {
-    if (ApiType == APIType::D3D || ApiType == APIType::Vulkan)
+    if (ApiType == APIType::D3D || !g_ActiveConfig.backend_info.bSupportsReversedDepthRange)
```

Comment:
Would it be cleaner to just set D3D to have no reversed depth range?  Maybe that's another PR..

Reply:
Yeah, it would. Dunno why I didn't think of this..

Reply:
@iwubcode That is already the case: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/VideoBackends/D3D/main.cpp#L76.

So yeah, the API check is redundant here.